react/components/GalleryItem.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react'
 import { ExtensionPoint } from 'vtex.render-runtime'
+import { usePixel } from 'vtex.pixel-manager/PixelContext'
```

Comment:
should this be part of this PR?

Reply:
when i made a rebase, this occurred, i don't know the reasons.